pkg/cmd/config-images.go

```diff
@@ -120,23 +120,29 @@ func listImages(opts *listImagesOpts) error {
 	if configErr == nil {
 		// Custom loading of the config is needed to avoid "normal" validation process, but we here don't care about
 		// validity of the config, the only part that's needed is `.RegistryConfiguration`
-		var conf kubeonev1beta1.KubeOneCluster
-		if err := yaml.Unmarshal(configBuf, &conf); err != nil {
-			return err
-		}
-
-		overRegGetter := images.WithOverwriteRegistryGetter(func() string {
-			if rc := conf.RegistryConfiguration; rc != nil {
-				return rc.OverwriteRegistry
+		func() {
+			var conf kubeonev1beta3.KubeOneCluster
+			if err := yaml.Unmarshal(configBuf, &conf); err != nil {
+				// we don't care if there was an error, just proceed like there was no manifest file given at all
```

Comment:
Why is this change needed?

Reply:
Initial idea was that "since we don't care if file exists, maybe we don't care about the unmarshal error ether". But yeah, those are different cases. Returned back as it was.